indico/web/forms/validators.py

```diff
@@ -379,6 +379,10 @@ class SoftLength(Length):
     where surrounding whitespace is stripped before validation.
     """
 
+    def __call__(self, form, field):
+        field.data = re.sub(r'(\r\n|\r)', '\n', field.data)
```

Comment:
I'm not sure I like the idea of modifying a field's data from within a validator... AFAIK that's what filters are meant for (defined on the field level - searching for `filters=[` will give you a few examples)

Reply:
Me neither, my initial idea wasn't to modify it within a validator but to make it ignore any double line-breaks. Filters definitely seem better suited here.

Reply:
The downside of filtering (instead of ignoring) is that we need to make sure this filter is applied anywhere we used a `TextArea` (or any of its superclasses) with a length validator, which isn't that easy to control. Even adding the filter by default in `IndicoMarkdownField` doesn't prevent this from happening with `TextArea` in the future. Should I just worry about `IndicoMarkdownField` for now?

Reply:
The alternative would be something like this:

```python
old_data = field.data
field.data = re.sub(...)
try:
    super()...
finally:
    field.data = old_data
```

Reply:
Seems fair, and simpler than cloning the field.